src/NBomber/NBomber.fsproj

```diff
@@ -109,6 +109,9 @@
         <EmbeddedResource Include="Resources\HtmlReport\assets\css\index.css" />
         <EmbeddedResource Include="Resources\HtmlReport\index.html" />
     </ItemGroup>
+    <ItemGroup>
+      <ProjectReference Include="..\NBomber.Contracts\NBomber.Contracts.fsproj" />
+    </ItemGroup>
```

Comment:
Not sure how the changes in the projects should be coordinated. Here I added a direct reference to `NBomber.Contracts.fsproj` for testing purposes.

Reply:
it's fine.
for dev we can keep it as you did